docs/source/services/timelock_service/product-changes.rst

```diff
@@ -0,0 +1,20 @@
+.. _product-changes:
+
+Developing a product to use the timelock service
+================================================
+ .. note::
+    We recommend only product instances deployed against Cassandra KVS to use the timelock server.
+
+All products deploying against the AtlasDB Timelock service should follow the following checklist.
+
+1. Ensure that the AtlasDB config contains :ref:`timelock-client-configuration`.
```

Comment:
I think that is clear in the link. I considered adding `and does not contain the
_Leader configuration_`. Two hyperlinks here are confusing and no hyperlink on the leader config is kind of inconsistent. 
http://palantir.github.io/atlasdb/html/configuration/external_timelock_service_configs/timelock_client_config.html  states the above very clearly.